python/pyspark/worker.py

```diff
@@ -254,7 +254,12 @@ def read_udfs(pickleSer, infile, eval_type):
         timezone = runner_conf.get("spark.sql.session.timeZone", None)
         safecheck = runner_conf.get("spark.sql.execution.pandas.arrowSafeTypeConversion",
                                     "false").lower() == 'true'
-        ser = ArrowStreamPandasSerializer(timezone, safecheck)
+        # NOTE: this is duplicated from wrap_grouped_map_pandas_udf
+        assign_cols_by_name = runner_conf.get(
```

Comment:
@BryanCutler, BTW, would you be willing to work on removing `spark.sql.legacy.execution.pandas.groupedMap.assignColumnsByName` in Spark 3.0? I think this makes the codes complicated.

Also, would you mind working on upgrading minimum Arrow to 0.12.0 as well, as we discussed? (Probably it better be asked to dev mailing list first to be 100% sure).

If you're currently busy, I will take one or both.

Reply:
Of course those should be separate JIRAs

Reply:
Yeah, definitely!  I could take those 2 tasks. I was thinking on holding off a little while to bump up the minimum Arrow version just to see if anything major came up in the meantime releases. 0.12.1 will be out in a couple days, but I don't think major bug fixes for us.  Maybe wait just a little bit longer?